docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-graph-security-api-source.md

```diff
@@ -102,15 +105,13 @@ To configure a Microsoft Graph Security API Source:
     ![Mircrosoft Graph Security API Source input.png](/img/send-data/Microsoft-Graph-Security-API-Source-input.png)
 
 1. (Optional) For **Source Category**, enter any string to tag the output collected from the Source. Category metadata is stored in a searchable field called `_sourceCategory`.
-1. **Forward to SIEM**. Check the checkbox to forward your data to Cloud SIEM Enterprise. If you click the checkbox, another option appears: Use **Dynamic Vendor and Product SIEM Metadata**.
-   *  If you don't checkmark **Use Dynamic Vendor and Product SIEM Metadata**, the metadata fields that identify vendor and product (`_siemVendor` and `_siemProduct`) will be set to _Security Graph API_.* If you do checkmark **Use Dynamic Vendor and Product SIEM Metadata**, CSE will retain the original product information. This is helpful when multiple data sources from multiple vendors and products are proxied through the Microsoft Security Graph API and you want the original vendor and product metadata information reflected in CSE. The table below shows the metadata fields that CSE sets.
-   <br/>
+1. **Forward to SIEM**. Check the checkbox to forward your data to Cloud SIEM Enterprise. When configured with the **Forward to SIEM** option the following metadata fields are set:
 
    | Field Name | Value   |
    | :-- | :-- | :-- |
    | _siemForward  | true |  
```

Comment:
```suggestion
   | _siemForward  | True |  
```

Reply:
as this is a boolean value, it should stay lowercased